test/functional/cmdLineTests/gcRegressionTests/playlist.xml

```diff
@@ -44,6 +44,36 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_GCRotatingVerboseLogTests</testCaseName>
+		<variations>
+			<variation>Mode110</variation>
+			<variation>Mode501</variation>
+			<variation>Mode551</variation>
+			<variation>Mode610</variation>
+		</variations>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/13891</comment>
+				<impl>ibm</impl>
+				<version>8</version>
+			</disable>
+		</disables>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -DTESTSJARPATH=$(Q)$(TEST_RESROOT)$(D)gcRegressionTests.jar$(Q) -DRESJAR=$(CMDLINETESTER_RESJAR) \
+		-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS)$(SQ) -jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)gcRotatingVerboseLogTests.xml$(Q) \
+		-explainExcludes -xids all,$(PLATFORM),$(VARIATION) -plats all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
```

Comment:
We need to use the exclude file as some tests do not run on wins. Please move GC rotating verbose exclude out of gcRegressionTests.xml and create a separate exclude file for  cmdLineTester_GCRotatingVerboseLogTests

Reply:
New exclude file has been added. Thanks.